Java/QuickSort .java

```diff
@@ -0,0 +1,62 @@
+public class QuickSort {  

```

Comment:
Please add time complexity

Reply:
sir, are you telling me to write the code for execution time???.......... I don't know code for time complexity.....